test02_cover_art.py

```diff
@@ -8,23 +8,34 @@
 
 class TestCover(unittest.TestCase):
 
-    def get_image(self):
+    def test_get_album_art(self):
         test_key = cover_art_api.key
-        test_album = "Come Fly With Me"
-        test_artist = "Frank Sinatra"
+        test_album = "Lemonade"
+        test_artist = "Beyonce"
         test_query = {"method" : "album.getinfo", "api_key" : test_key, "artist" : test_artist, "album" : test_album, "format" : "json"}
-        test_cover_response = requests.get(cover_art_api.url, params=test_query).json()
-        self.assertEqual(cover_art_api.cover_response, self.test_cover_response)
-    
-    def test_image_download(self):
-        self.images = self.test_cover_response.get('album').get('image')
-        self.assertEqual(self.test_cover_response, cover_art_api.cover_response)
+        test_query_results = cover_art_api.get_album_art(test_album, test_artist)
+        self.assertEqual(test_query , test_query_results)
+
+    def test_get_image_by_size(self):
```

Comment:
What is this testing? You have both the function call `cover_art_api.get_image_by_size("images" , "size")` and the implementation, lines 23 - 26 in the same test.